pipeline-scripts/buildlib.groovy

```diff
@@ -164,7 +164,9 @@ def setup_venv(use_python38=false) {
         commonlib.shell(script: "rm -rf art-tools/elliott ; cd art-tools; git clone https://github.com/${where[0]}/elliott.git; cd elliott; git checkout ${where[1]}")
```

Comment:
doozer_commit and elliott_commit params would stop working, and I think we want to have art_tools_commit in place of them (see promote / prepare release / build-sync accepting them)

Reply:
I see, let me see if I can understand whats happening. But when we ran the prepare release test job as dry run, it [worked](https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/hack/job/ashwin-aos-cd-jobs/job/build%252Fprepare-release/2/console).

Reply:
For prepare-release the change was recent and the job run is older (13 days ago) so it doesn't have the elliott_commit param. To see it break you would have to run it with a `elliott_commit` value like `ashwindasr@some_branch_name`

Reply:
Understood, will start working on those as well and will add to this PR